backend/lib/models.ml

```diff
@@ -28,43 +41,58 @@ module Benchmark = struct
 
   let pull_number self = self.pull_number
 
-  let name self = self.name
+  let test_name self = self.test_name
 
-  let data self = self.data
+  let benchmark_name self = self.benchmark_name
+
+  let metrics self = self.metrics
 
   let pp =
     let open Fmt.Dump in
     record
       [
         field "run_at" run_at Ptime.pp;
         field "duration" duration Ptime.Span.pp;
-        field "repo_id" duration Ptime.Span.pp;
+        field "repo_id" repo_id Fmt.(pair ~sep:(Fmt.any "/") string string);
```

Comment:
There was a slight mistake actually here

Reply:
Nice catch! I haven't used the printing functions yet so didn't notice the problem.